src/openforms/emails/utils.py

```diff
@@ -100,3 +103,23 @@ def render_email_template(
         backend=openforms_backend,
         disable_autoescape=disable_autoescape,
     )
+
+
+def collect_failed_emails(since: datetime) -> list[dict[str, str] | None]:
+    logs = TimelineLogProxy.objects.filter(
+        timestamp__gt=since,
+        extra_data__status=Message.STATUS_FAILED,
+        extra_data__include_in_daily_digest=True,
+    ).distinct("content_type", "extra_data__status", "extra_data__event")
```

Comment:
The docs [say](https://docs.djangoproject.com/en/5.0/ref/models/querysets/#django.db.models.query.QuerySet.distinct):

> When you specify field names, you must provide an `order_by()` in the `QuerySet`, and the fields in `order_by()` must start with the fields in `distinct()`, in the same order.

I see the code was moved, so it was probably already working. Is there a need to add an `order_by` call here?

Reply:
As it is done for `collect_registrations_failures`, maybe we should also filter on `extra_data__log_event="email_status_change"`? It might be that other logs are using `4` (the value of `Message.STATUS_FAILED`) as a value for `extra_data["status"]`.

Reply:
Ah yes I meant the template field indeed

Reply:
w/r to the order_by, it's most likely implicit on the model itself. Postgres complains/crashes hard if it's misaligned, so the existing test coverage helps enforce this.